pkg/keccak/Makefile

```diff
@@ -0,0 +1,24 @@
+PKG_NAME    := keccak
+PKG_URL     := https://github.com/gvanas/KeccakCodePackage
+PKG_VERSION := bc592d1780a6b3204c26c6f872552ace8fc6fd65
+
+ifeq ($(CPU),native)
+  CPU_ARCHIVE_NAME := Native
+else
+ifeq ($(CPU_ARCH),$(filter $(CPU_ARCH), cortex-m0 cortex-m0plus))
```

Comment:
the previous CPU_ARCH is now called CPU_CORE. there's now a CPU_ARCH containing "armv6m" or "armv7m", you might want to change the list accordingly.

Reply:
Like this?

```Makefile
ifeq ($(CPU),native)
  CPU_ARCHIVE_NAME := Native
else
ifeq ($(CPU_ARCH),armv6m)
  CPU_ARCHIVE_NAME := ARMv6M
else
ifeq ($(CPU_ARCH),armv7m)
  CPU_ARCHIVE_NAME := ARMv7M
endif
endif
endif
```

Reply:
The rename `CPU_ARCH` to `CPU_CORE` is missing.

Reply:
The code that is shown at the top of the conversation is outdated, it now looks like what I wrote in my comment above yours, so I use the new `CPU_ARCH` instead of the old `CPU_ARCH` / now `CPU_CORE`. Or is that not a good idea?

Reply:
Ah sorry, I misunderstood @kaspar030's comment that the variable was just renamed and there is no `CPU_ARCH` anymore ^^"